backend/src/main/resources/application.yml

```diff
@@ -57,4 +57,4 @@ spring:
   flyway.locations: classpath:db/migration/h2
 logging.level:
   org.hibernate.SQL: DEBUG
-  org.hibernate.type.descriptor.sql.BasicBinder: TRACE
```

Comment:
is this intended?

Reply:
No, seems like it was deleted by IDEA unused code remover